ci/test_wheel_cuopt.sh

```
@@ -8,6 +8,7 @@ set -euo pipefail
 # sets up a constraints file for 'pip' and puts its location in an exported variable PIP_EXPORT,
 # so those constraints will affect all future 'pip install' calls
 source rapids-init-pip
+source ./ci/use_wheels_from_prs.sh
```

Shellcheck (0.11.0)

[info] Not following: ./ci/use_wheels_from_prs.sh was not specified as input (see shellcheck -x).

(SC1091)